### DISCUSSION TITLE
Impact of the version of CP2K on the initial initial dataset used by deepmd-kit

### DISCUSSION THREAD

**Discussion by hanrm94** [deepmodeling/deepmd-kit#4094] [Q&A]
Can the same initial dataset used by deepmd-kit come from the output files of different versions of cp2k's calculations? The versions of cp2k are 7.1 and 2024.1.
Also, can cp2kdata be converted from cp2k2024.1 to deepmd-kit format?

**Comment by njzjz** [comment]
fyi @robinzyb

**Comment by robinzyb** [comment]
1. In general, the answer is yes because cp2k has a testsuit for checking numerical accuracy after the installation. But I would recommend you to recalculate one or two old data/structures using v 2024.1 to see whether it gives same results
2. yes, cp2kdata can convert cp2k 2024.1 data to dpmd format
